LiskHQ/lisk-sdk
# Integration tests failing on development - Closes #3794

### What was the problem?
Wrong expectation on normalizeBlock validations test. 
If transaction type is not present it immediately throws required property error without evaluating `Invalid type`

### How did I fix it?
By removing the invalid expectation

### How to test it?
`npm run mocha:integration --prefix ./framework test/mocha/integration/blocks/verify/verify.js`

### Review checklist

* The PR resolves #3794
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
